steps/dns/steps.go

```diff
@@ -38,8 +40,17 @@ func (s Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioConte
 	ctx = InitializeContext(ctx)
 	session := GetSession(ctx)
 
+	// Set the DNS responses timeout
+	// Set if indicated in environment parameter: CONF:timeout
+	// If not indicated, the default value is kept in 2 seconds (mikeg library)
+
+	if timeout, err := strconv.ParseInt(golium.ValueAsString(ctx, "[CONF:timeout]"), 10, 64); err == nil {
```

Comment:
I guess that it's more natural to use a step to configure the timeout (I guess that it's not a usual step that you require).
Something like: 
`^a DNS timeout of "(\d+)" milliseconds$`

You are linking the steps with a specific configuration setting, that I don't think it's recommendable.

Other option is to including these settings, as default values, in the Steps struct. However, I would recommend the simple approach of creating another step to configure the timeout.